src/features/customerView/openInvoicesCard/OpenInvoicesCard.tsx

```diff
@@ -94,9 +95,14 @@ const OpenInvoicesCard = ({ invoices, handleShowInvoice }: OpenInvoicesCardProps
         </Section>
         {isBerthInvoice(invoice) && <BerthContractDetails leaseId={invoice.leaseId} />}
         {isWinterStorageInvoice(invoice) && <WinterStorageContractDetails leaseId={invoice.leaseId} />}
-        <Button variant="secondary" theme="coat" onClick={() => handleShowInvoice(invoice)} className={styles.button}>
-          {t('customerView.customerInvoice.showInvoice')}
-        </Button>
+        <div className={styles.buttons}>
+          <Button variant="secondary" theme="coat" onClick={() => handleShowInvoice(invoice)} className={styles.button}>
+            {t('customerView.customerInvoice.showInvoice')}
+          </Button>
+          <Button variant="primary" theme="coat" onClick={() => handleResendInvoice(invoice)} className={styles.button}>
```

Comment:
We have a wrapper for HDS `Button` which already specifies its `theme`.
https://github.com/City-of-Helsinki/berth-reservations-admin/blob/a0f300638338cd5c9901e410e4756c12c692e293/src/common/button/Button.tsx#L5